packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -10,11 +10,11 @@
             :tabindex="isBelowMid && !navIsOpen ? -1 : 0"
             button-type="icon"
             class="c-nav-list-text c-countrySelector-btn"
-            :aria-expanded="!isBelowMid && countrySelectorIsOpen ? 'true' : 'false'"
-            :aria-haspopup="isBelowMid ? false : true"
+            :aria-expanded="countrySelectorIsOpenOnDesktopView ? 'true' : 'false'"
```

Comment:
💯

Reply:
Would the following work?
`:aria-expanded="countrySelectorIsOpenOnDesktopView`

Reply:
Does the value not have to explicitly be "true" or "false"?

Reply:
It does IIRC - the element needs to have the aria attribute present at element intiaisation for screen readers to be aware of it and react to it changing, and as per https://vuejs.org/v2/guide/syntax.html#Attributes :

> In the case of boolean attributes, where their mere existence implies true, v-bind works a little differently. In this example:
> 
>       <button v-bind:disabled="isButtonDisabled">Button</button>
>
> If isButtonDisabled has the value of null, undefined, or false, **the disabled attribute will not even be included** in the rendered <button> element.

Reply:
Could change this line to `:aria-expanded="countrySelectorIsOpenOnDesktopView.toString()` instead then?

Reply:
Or just leave it as it was 😄